tests/testCompoundInstanceCatalog.py

```diff
@@ -288,6 +288,60 @@ def testCompoundCatalog(self):
         if os.path.exists(fileName):
             os.unlink(fileName)
 
+    def testSharedVariables(self):
```

Comment:
Ok, so this is to test the property mentioned in the note added compoundInstanceCatalog.py. This makes the whole note much clearer. Was this common set of transformations the motivation behind adding the functionality? Are there any other useful examples? Just wondering, I like it as an addition now that I understand why it might be used in at least one case.